src/path-planner.cc

```diff
@@ -149,7 +163,8 @@ namespace hpp {
 	  PathValidationReportPtr_t report;
           bool pathValid = pathValidation->validate (projPath, false, validPath,
 						     report);
-          if (pathValid && validPath->length() > 0) {
+          if (pathValid && validPath->timeRange ().second !=
+              path->timeRange ().first) {
```

Comment:
Is there a reason for this change ?

Reply:
No, I probably took this line by mistake from an older commit. Fixed in 8891c44